app-sso/getting-started/appsso-overview.hbs.md

```diff
@@ -5,9 +5,9 @@ Single Sign-On (commonly called AppSSO).
 
 Use this topic to learn how to:
 
-1. [Set up your first authorization server](provision-auth-server.hbs.md).
-1. [Provision a ClientRegistration](client-registration.hbs.md).
-1. [Deploy an application](application.hbs.md) that uses the provisioned ClientRegistration to enable SSO.
+1. [Set up your first simplistic authorization server](./provision-auth-server.hbs.md).
+1. [Claim credentials to connect your workload](./claim-credentials.hbs.md)
```

Comment:
**[issue, non-blocking]:** I find the term "connect" to be unspecific. How about simply "claim credentials"?

Reply:
Sounds good!